_topic_map.yml

```diff
@@ -1542,7 +1552,7 @@ Topics:
 ---
 Name: Serverless applications
 Dir: serverless
-Distros: openshift-enterprise,openshift-webscale
+Distros: openshift-enterprise,openshift-webscale,openshift-aro
```

Comment:
Some more 404s: Serverless -> Installing OpenShift Serverless:
- in the important note
- Cluster sizing requirements -> "Creating MachineSets" link

Serverless applications -> Removing OpenShift Serverless -> Removing the OpenShift Serverless Operator -> "deleting Operators from a cluster."
`The requested URL /klamenzo/aro-4x-preview/operators/olm-deleting-operators-from-cluster.html was not found on this server.`